docs/getting-started/dockstore-workflows.rst

```diff
@@ -47,149 +48,125 @@ below:
-where the files are likely to be present.
-
-You can set a default test parameter file for a workflow, which defaults
-to ``/test.json``. Whenever a new version is added, Dockstore will look
-for a file matching the default test parameter in the corresponding Git
-repository, and add it if it exists. Updating the default test parameter
-file path for an existing workflow will be propagated to any versions
-that have not been edited.
-
-.. raw:: html
-
-   <!-- this following markdown link/anchor does not seem to work properly -->
-
-The second step is to log in to the Dockstore. Make sure that you
-properly link Bitbucket and/or GitLab to your account if you are using
-workflows hosted on Bitbucket/Gitlab. After successfully linking your
-GitHub, Bitbucket and GitLab credentials (if applicable), you will be
-able to refresh your account on Dockstore and list your available repos
-on GitHub, Bitbucket and GitLab.
-
-.. figure:: /assets/images/docs/workflow_ui.png
-   :alt: My Workflows
-
-   My Workflows
-
-The above image shows the general structure of the UI that you should
-see after visiting "My Workflows." You can hit "Refresh All Workflows"
-in order to update information on published workflows or create new
-stubs for repos that you are about to publish. Workflows are a bit
-different from tools in that we create a stub entry for all your GitHub
-repos (and Bitbucket/GitLab repos). You can then promote these to full
-entries by refreshing, publishing and editing them. It is only at this
-point that the Dockstore will reach out and populate information such as
-available tags and source files. Workflows are handled differently from
-Tools in this regard since users may often have many more tags and repos
-on GitHub than Docker images on Quay.io.
+Prepare Your Workflow for Dockstore
+-----------------------------------
+This tutorial does not go through the creation of a workflow and its
+registration to GitHub, Bitbucket or GitLab. It assumes that you already
+have a repository which contains a workflow and are now trying to register
+it in Dockstore. There are some ways to make the registration process more
+seamless.
+
+- For your primary workflow descriptor, use the filename ``Dockstore.<ext>``
+  at the root of your repository
+- For your test parameter files, use the filename ``test.json`` at the root
+  of your repository
+- There should be one workflow per repository
+
+By default, Dockstore will search the root of your repository for workflow
+related files. Following the above tips will help streamline the registration
+process, though you can still register workflows with non-standard format.
 
 Register Your Workflow in Dockstore
 -----------------------------------
+There are a variety of ways to get your workflows into Dockstore. There are two
+types of quick registration and one type of manual registration. These quick
+methods create workflow stubs, which are simply workflows that have not yet
```

Comment:
I thought you got rid of stubs?

Reply:
Replied to this in a general comment below.